.eslintrc.js

```diff
@@ -53,6 +54,10 @@ module.exports = {
     },
   },
   overrides: [
+    {
+      files: ['packages/**/*.ts'],
+      extends: ['plugin:@typescript-eslint/recommended'],
+    },
```

Comment:
```suggestion
    {
      files: ['packages/**/*.ts'],
      extends: ['plugin:@typescript-eslint/recommended'],
      parserOptions: {
        // "project" is required for rules requiring type information
        project: true, // "true" finds the tsconfig.json nearest each source file
      },
    },
```

could also be a good idea for future `rules` we add here

Reply:
@leotm Per [the docs](https://typescript-eslint.io/linting/typed-linting/monorepos#one-tsconfigjson-per-package-and-an-optional-one-in-the-root) it appears that we'll need to specify the paths via a glob or otherwise instead of use `true` here